### ISSUE TITLE
linter-gcc only lints CMakeLists.txt, but not my .cpp files

### ISSUE THREAD

**Issue by johannesstricker** [hebaishi/linter-gcc#127]
## Checklist
* [x] Check that you have the GNU C Compiler installed by typing ```gcc --version``` into a terminal.
* [x] If you don't have the ```linter``` package installed, either update to the latest version of ```linter-gcc``` or install linter directly with ```apm install linter```.
* [x] Ensure that you are using a valid grammar - either C/C++/C++14
* [x] Ensure that you have read all the relevant information in the [wiki](https://github.com/hebaishi/linter-gcc/wiki), particularly the [Common Issues](https://github.com/hebaishi/linter-gcc/wiki/Common-Issues) page.

## Issue (including debugging messages):
It seems that linter-gcc tries to lint my CMakeLists.txt files. I'm not sure if this is an error on my side, because of how I set up my build system. 
However if I add double semicolon at the end of the line or use an undefined variable in one of my .cpp files the linter does not complain. I tried copying the linter-gcc command into the terminal but that only shows the CMakeLists errors as well.
